.github/workflows/master.yml

```diff
@@ -0,0 +1,147 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  python35:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.3.4
+      PANDAS_VERSION: 0.23.4
+      PYARROW_VERSION: 0.10.0
+      PYTHON_EXECUTABLE: xvfb-run python
+      KOALAS_USAGE_LOGGER: databricks.koalas.usage_logging.usage_logger
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    # Setup Python 3.5 via `apt-get install` since the dafault Python3.5 from `actions/setup-python`
+    # seems to have some problem with Tkinter, so we should manually install the python3.5-tk.
+    # for this, we should use the Python we manually installed, not the default one from `actions/setup-python`
+    - name: Setup Python 3.5
+      run: |
+        sudo add-apt-repository ppa:deadsnakes/ppa
+        sudo apt-get update
+        sudo apt-get install tk-dev python3.5-tk python3 python3.5 python-pip python3-pip
```

Comment:
Why do we need to install `python3` and `python-pip`?  Also, does `python3-pip` guarantees to install Python 3.5's PIP? If we're not sure,  you can just try `/usr/bin/python3.5 -m ensurepip` instead.

Reply:
Thanks for the comment!

it was fixed by adding the path of packages are installed by PyPI to the `PATH` using `add-path`.